papers/joss/paper.md

```diff
@@ -21,9 +21,12 @@ bibliography: paper.bib
 
 A common component of frameworks for building robust and accurate learning models is a utility for performing sets of perturbations on the input data. In machine vision, for example, rotating, cropping, stretching, warping, and flipping training images have all been shown to increase model accuracy, and reduce certain kinds of overfitting [@chen2019invariance]. Popular machine vision libraries like PyTorch, Tensorflow, and FastAI all have built-in utilities for performing these transformations [@tensorflow2015;@NEURIPS2019_9015;@howard2020fastai].
 
-In the domain of NLP, there is an analogous set of transformations that have been shown to increase model accuracy, or make models more robust in the face of adversarial inputs. These include adding superfluous whitespace, swapping an individual word for a synonym, and appending the word "love" to the end of a sentence (this last one, in an attempt to fool sentiment classifiers) [@zhang2015characterlevel;@coulombe2018text;@grndahl2018need]. There are a small number of libraries that contain text augmentation functions, for example [noisemix](https://github.com/noisemix/noisemix) and [EDA](https://github.com/jasonwei20/eda_nlp) [@wei2019eda], but these tend to be associated with conference papers and have little documentation, or only implement a small subset of common transforms.
+In the domain of NLP, there is an analogous set of transformations that have been shown to increase model accuracy. Common transformations include adding superfluous whitespace, swapping an individual word for a synonym, replacing a word with a misspelled version, and round-tripping a sentence through a translation model into another language and back again [@zhang2015characterlevel;@coulombe2018text]. Some transforms, like misspellings or synonyms, attempt to directly model an unobserved but common change in the input features. Others, like adding a space in the middle of a word, act similarly to dropout for word-level models, in that they effectively the word as an input (e.g. the two halves of the word are unlikely to be present in the vocabulary).
```

Comment:
Missing a word after "...effectively the word..."

Reply:
🤦‍♀️ I accidentally the words a lot